.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: node_js
```

Comment:
I think you might not need the new jobs stuff since there is not a lot of stuff to do on this repo yet. For NPM deploy you just need

before_deploy:
- npm --no-git-tag-version version 0.1.0-prerelease.$(date +%Y%m%d%H%M%S)
- git config --global user.email $(git log --pretty=format:"%ae" -n1)
- git config --global user.name $(git log --pretty=format:"%an" -n1)
deploy:
- provider: npm
  on:
    branch: develop
  skip_cleanup: true
  email: $NPM_EMAIL
  api_key: $NPM_TOKEN

You can generate a token at https://www.npmjs.com/settings/tokens

I also didn't need the script: npm run $NPM_SCRIPT for scratch-paint

Reply:
Unless it causes problems I think I'd rather use this more-general form for now, if only for the sake of consistency with other repos.